src/sasquatchbackpack/scripts/usgs.py

```diff
@@ -26,9 +28,11 @@ def search_api(
         max_magnitude (int, optional): maximum earthquake magnitude.
             Defaults to 10.
     """
-    current_dt = datetime.utcnow()
+    # Linting bypassed, as at the time of writing Libcomcat breaks if provided
+    # with a timezone-aware datetime object
+    current_dt = datetime.now(None)  # noqa: DTZ005
```

Comment:
So ruff provides the right answer here:
```
    current_dt = datetime.now(UTC)
```
See [datetime documentation](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) and the deprecation notice of `datetime.utcnow()` in 3.12